config/routes.rb

```diff
@@ -20,6 +20,8 @@
     root to: 'candidates/home#index'
   end
 
+  get "/candidates/what_to_expect", to: 'candidates/home#what_to_expect'
```

Comment:
```suggestion
  get "/candidates/what-to-expect", to: 'candidates/home#what_to_expect', as: "what_to_expect"
```

Reply:
you might not need the `as` ☝🏽  - but most (all?) of the other paths use dashes instead of underscores

Reply:
Ah you're right - confused with GIT 🙈